integrations/google_genai/src/haystack_integrations/components/generators/google_genai/chat/chat_generator.py

```diff
@@ -150,20 +158,23 @@ def _convert_google_genai_response_to_chatmessage(response: types.GenerateConten
     if response.candidates:
         candidate = response.candidates[0]
         finish_reason = getattr(candidate, "finish_reason", None)
-        for part in candidate.content.parts:
-            if part.text is not None:
-                text_parts.append(part.text)
-            if part.function_call is not None:
-                tool_call = ToolCall(
-                    tool_name=part.function_call.name,
-                    arguments=dict(part.function_call.args) if part.function_call.args else {},
-                    id=part.function_call.id,
-                )
-                tool_calls.append(tool_call)
+        if candidate.content is not None and candidate.content.parts is not None:
```

Comment:
Interesting, I always try to make these truthy as it is less cognitive load for me and perhaps even more idiomatic, no?